custom/icds_reports/const.py

```diff
@@ -10,6 +11,8 @@
     VHSNDMonthlyReport,
 )
 
+SQL_TEMPLATES_ROOT = os.path.join(os.path.dirname(__file__), "migrations", "sql_templates")
```

Comment:
nit: `SQL_MIGRATIONS_TEMPLATES_ROOT` or something like that that conveys its for migration.

Reply:
thanks for the suggestion